modules/BTBidAdapter.js

```diff
@@ -89,14 +104,25 @@ function bidResponse(buildBidResponse, bid, context) {
  * @returns {boolean} True if the bid request is valid, false otherwise.
  */
 function isBidRequestValid(bid) {
-  const { ab, siteId } = bid.params;
+  const { blockthrough } = bid.params;
+  if (!isPlainObject(blockthrough) || !Object.keys(blockthrough).length) {
+    logWarn(
+      'BT Bid Adapter: a object type "blockthrough" with site ids and adblock data must be provided.'
+    );
+    return false;
+  }
+
+  if (!blockthrough.orgID || !isStr(blockthrough.orgID)) {
+    logWarn('BT Bid Adapter: a string type "orgID" must be provided.');
+    return false;
+  }
 
-  if (!siteId || !isStr(siteId)) {
-    logWarn('BT Bid Adapter: a string type "siteId" must be provided.');
+  if (!blockthrough.websiteID || !isStr(blockthrough.websiteID)) {
+    logWarn('BT Bid Adapter: a string type "websiteID" must be provided.');
     return false;
   }
 
-  if (!isBoolean(ab)) {
+  if (!isBoolean(blockthrough.ab)) {
```

Comment:
Would it be better to mention that these params have to be provided in the `blockthrough` level in the log warnings?